### PR TITLE
Added logging/handling around session destruction.

### PR DESCRIPTION
**Pull request by davespanton**
In order to investigate errors in parsing data from DynamoDB.

### CODE REVIEW

**Comment by antstorm** [comment]
Looking good!

File: lib/connect-dynamodb.js [hunk 1]
```diff
@@ -208,10 +210,17 @@ module.exports = function (connect) {
         function destroyDataAt(index) {
             if (data.Count > 0 && index < data.Count) {
                 var sid = data.Items[index].id.S;
```
Comment (crebbo) [inline lib/connect-dynamodb.js]: could this not throw an error if it is `data.Items[index]` or doesn't have `id` on it, is that possible?

Comment (crebbo) [inline lib/connect-dynamodb.js]: @davespanton ?

Comment (davespanton) [inline lib/connect-dynamodb.js]: In theory yes. The same could be said of all the properties accessed from the returned data on any call via the aws-sdk. The changes here only only protect against the specific incident we saw. How much protection we want from potentially unreliable data is another question.

Comment (crebbo) [inline lib/connect-dynamodb.js]: ok, that's good enough I guess

**Pull request merged**